CMakeLists.txt

```diff
@@ -71,4 +71,12 @@ add_library(enet STATIC
 
 if (MINGW)
     target_link_libraries(enet winmm ws2_32)
-endif()
\ No newline at end of file
+endif()
+
+install(TARGETS enet
+    RUNTIME DESTINATION bin
+    ARCHIVE DESTINATION lib/static
+    LIBRARY DESTINATION lib)
+
+install(DIRECTORY include/
```

Comment:
+1 to this commit in general!

after some research on my own I believe the "accepted" way to include headers is:

set_target_properties(enet PROPERTIES PUBLIC_HEADER "${INCLUDE_FILES}")
install(
	TARGETS enet
	LIBRARY DESTINATION lib
	PUBLIC_HEADER DESTINATION include/enet
)

Its cmake so maybe there is some side effect to PUBLIC_HEADERS or some advantage to your method that I am not aware of.